biggraphite/drivers/elasticsearch.py

```diff
@@ -958,14 +997,18 @@ def __update_document(self, data, index, document_id):
             ignore=[404, 409],
         )
 
+    def _create_directory_search_query(
```

Comment:
Docs what does "index" mean in the context of this function? does it mean that the directory is or is not indexed? consider `is_indexed` if it is a boolean also.

Reply:
index is the index used for the search, if not present then it will be set by the method

Reply:
FAR: It still might be worth documenting the parameters.